frontend/layouts/static-page/header/component.tsx

```diff
@@ -1,5 +1,6 @@
-import React, { useState, useCallback } from 'react';
+import React, { useState, useCallback, useEffect } from 'react';
```

Comment:
These changes come from https://github.com/Vizzuality/heco-invest/pull/89 right? Can you remove the commits associated with this PR?